performance-tools/benchmark.md

```diff
@@ -0,0 +1,38 @@
+# Computer Vision Pipeline Benchmarking
+
+The provided Python-based script works with Docker Compose to get pipeline performance 
+metrics like video processing in frames-per-second (FPS), memory usage, power
+consumption, and so on.
+
+# Prerequisites 
+
+- Python environment v3.12.2
+  - could use [Miniconda](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-python.html) and create a [Python 3.12.2 env](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-python.html)
+- Python packages listed in requirements.txt
+- Docker
+- Docker Compose
+- Make
+- Git
+- Code from [Retail Use Cases Repo](https://github.com/intel-retail/retail-use-cases) and [Performance Tools Repo](https://github.com/intel-retail/performance-tools)
```

Comment:
This one is interesting and bit more into it: the retail use-case repo has the submodules of performance tools. So if we have the retail use-cases repo cloned and git submodule initialized; then this automatically met.  In terms of documentation, maybe worth to note that in the retail-use-cases repo sub-module initialized, the user can use `make update-submodules` so that this prerequisites can be met for the sake of user's awareness on this.

Reply:
ok. I can update the docs to reflect this.